MIP40/mip40.md

```diff
@@ -91,6 +91,14 @@ Once a MIP40c3-SP subproposal passes, the Governance Facilitators or the MIP Edi
 
 This is a technical process component that can have on-chain effects that alter the Maker Protocol's state. When subproposals generated from this component reach the executive vote stage of the MIPs cycle, the executive vote must include a technical state change to authorize the budget implementations specified in the subproposal to draw funds from the Maker Protocol.
 
+A budget modification proposal for a Core Unit made by anyone other than a member or facilitator of that Core Unit may only be to reduce budget to 0, and may not be voted upon until the earliest of:
+
+- All active DAI budgets to the Core Unit have been fully disbursed.
+- Two months from RFC submission.
```

Comment:
From the RFC submission of the budget that's being set to zero? Given that MIP40c3 SPs have a one-month Feedback Period and that Ratification Polls go up one week after the opening of the Formal Submission Window and that they run for two weeks, that's pretty much as soon as it's been approved, isn't it?

Reply:
I think this is specifically giving teams a little more runway. So we have to figure out if it's modifying the governance cycle for this specific use case (unlikely) or something akin to keeping the RFC period open longer. My hunch is the latter.

Because of this [post mortem thread](https://forum.makerdao.com/t/content-production-core-unit-offboarding-post-mortem/13756) we should consider an additional change. 

> - All active DAI budgets to the Core Unit have been fully disbursed _or returned to the pause proxy._